tICA/scripts/tICAMakeCleanLongitudinalTemplate.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+set -eu
+
+pipedirguessed=0
+if [[ "${HCPPIPEDIR:-}" == "" ]]
+then
+    pipedirguessed=1
+    #fix this if the script is more than one level below HCPPIPEDIR
+    export HCPPIPEDIR="$(dirname -- "$0")/.."
+fi
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/tempfiles.shlib" "$@"
+
+#description to use in usage - syntax of parameters is now explained automatically
+opts_SetScriptDescription "creates tICA cleaned data for all longitudinal sessions in template directory"
+
+opts_AddMandatory '--study-folder' 'StudyFolder' 'path' "folder that contains all sessions"
+opts_AddMandatory '--subject' 'Subject' 'str' 'longitudinal subject ID'
+opts_AddMandatory '--session-list' 'SesslistRaw' 'HCA6002236_V1_MR@HCA6002236_V2_MR...' "list of longitudinal timepoint/session IDs separated by @s."
+opts_AddMandatory '--template-long' 'TemplateLong' 'ID' 'Longitudinal template ID'
+opts_AddMandatory '--extract-fmri-name-list' 'concatNamesToUse' 'name@name@name...' "list of fMRI run names to concatenate into the --extract-fmri-out output after tICA cleanup"
+opts_AddMandatory '--highpass' 'HighPass' 'hp value' 'High pass used with these data'
+opts_AddMandatory '--extract-fmri-name' 'extractNameOut' 'name' "fMRI name for concatenated extracted runs.
+Must match the one used in cortical registration."
+opts_AddOptional  '--reg-name' 'RegName' 'registration algorithm' "Cortical registration algorithm used [MSMAll]" "MSMAll"
+opts_AddOptional  '--fmri-name-concat-all' 'extractNameAll' 'name' "Concatenated output run label. If specified, must match the one used in multi-run FIX and cortical registration." ""
+opts_AddOptional  '--fmri-names' 'fMRINames' 'rfMRI_REST1_LR@rfMRI_REST1_RL...' "list of all fmri run names separated by @. Required with --extract-fmri-out-all" ""
```

Comment:
`--extract-fmri-out-all` doesn't exist, is this what you meant?  Also, you don't need to specify the empty string as the default for optionals (unless there are compatibility switches).
```suggestion
opts_AddOptional  '--fmri-name-concat-all' 'extractNameAll' 'name' "Concatenated output run label. If specified, must match the one used in multi-run FIX and cortical registration.  Requires --fmri-names."
opts_AddOptional  '--fmri-names' 'fMRINames' 'rfMRI_REST1_LR@rfMRI_REST1_RL...' "when using --fmri-name-concat-all, specify the list of MR FIX run names separated by @ here"
```